azext_iot/tests/test_iot_ext_int.py

```diff
@@ -40,6 +43,8 @@
 class TestIoTHub(LiveScenarioTest):
     def setUp(self):
         self._entity_names = None
+        self.cmd('az iot hub consumer-group create --hub-name {} --resource-group {} --name {}'.format(LIVE_HUB, LIVE_RG, LIVE_CONSUMER_GROUP1), checks=[self.check('name', LIVE_CONSUMER_GROUP1)])
```

Comment:
Does an exception get raised if the 'test1', 'test2' consumer-groups already exist?